PrintNodePrintJob.cs

```diff
@@ -136,6 +147,18 @@ public sealed class PrintNodePrintJob
             var list = JsonConvert.DeserializeObject<IEnumerable<IEnumerable<PrintNodePrintJobState>>>(response);
 
             return list.FirstOrDefault();
+        }
+
+		 public async Task<long> Print()

```

Comment:
Is this necessary when you can use PrintNodePrinter.AddPrintJob()?

Reply:
I wanted to be able to print a job without having to query PrintNode for
the printer first.

I already had the ID from the Print Dialogue I had shown the user.